commons/ava_testsuite/rpc_manager.go

```diff
@@ -0,0 +1,147 @@
+package ava_testsuite
+
+import (
+	"github.com/kurtosis-tech/ava-e2e-tests/commons/ava_default_testnet"
+	"github.com/kurtosis-tech/ava-e2e-tests/gecko_client"
+	"github.com/palantir/stacktrace"
+	"github.com/sirupsen/logrus"
+	"time"
+)
+
+const (
+	GENESIS_USERNAME = "genesis"
+	GENESIS_PASSWORD = "genesis34!23"
+	ACCEPTED_STATUS = "Accepted"
+)
+
+type RpcManager struct {
+	client *gecko_client.GeckoClient
+	testNet *ava_default_testnet.TestNet
+	rpcUser *RpcUser
+}
+
+func NewRpcManager(
+		client *gecko_client.GeckoClient,
+		testNet *ava_default_testnet.TestNet,
+		username string,
+		password string) *RpcManager {
+	return &RpcManager{
+		client: client,
+		testNet: testNet,
+		rpcUser: NewRpcUser(username, password),
+	}
+}
+
+type RpcUser struct {
+	username string
+	password string
+	payerNonce int
+}
+
+func NewRpcUser(username string, password string) *RpcUser {
+	return &RpcUser{username: username, password: password, payerNonce: 0}
+}
+
+func (rpcUser RpcUser) incrementNonce() int {
```

Comment:
wat dis do?

Reply:
I'm going to refactor the nonce increments so they're tied to accounts

Reply:
done